recipe/meta.yaml

```diff
@@ -18,13 +18,13 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3
```

Comment:
ah. take these selectors off, as the 
```
skip: True  # [py2k]
``` 
is sufficient. And will keep the windows paths from going :monkey: